src/AggregateFunctions/QuantileDD.h

```diff
@@ -33,14 +33,14 @@ namespace DB
 */
 
 template <typename Value>
-class QuantileDDSketch
+class QuantileDD
```

Comment:
We should remove this header altogether and move this into `AggregateFunctionQuantileDD.cpp`, as we do with other aggregations.